package.json

```diff
@@ -86,6 +86,7 @@
     "3box/**/libp2p-crypto/node-forge": "^0.10.0",
     "3box/**/libp2p-keychain/node-forge": "^0.10.0",
     "analytics-node/axios": "^0.21.2",
+    "analytics-node/axios/follow-redirects": "^1.14.7",
```

Comment:
This was an in-range update, so a resolution is not required. Just the lockfile update is enough